### ISSUE TITLE
Error when running chatgpt-md-translator

### ISSUE THREAD

**Issue by deining** [smikitky/chatgpt-md-translator#21]
I tried to install `chatgpt-md-translator` on my Windows box and later on on Ubuntu Linux box, both installations failed with more or less similar errors. That's what I'm getting when invoking the translator from my Ubuntu shell: 

```
$ chatgpt-md-translator
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/usr/lib/node_modules/chatgpt-md-translator/lib/utils/error-utils' imported from /usr/lib/node_modules/chatgpt-md-translator/lib/utils/fs-utils.js
    at new NodeError (node:internal/errors:405:5)
    at finalizeResolution (node:internal/modules/esm/resolve:226:11)
    at moduleResolve (node:internal/modules/esm/resolve:838:10)
    at defaultResolve (node:internal/modules/esm/resolve:1036:11)
    at DefaultModuleLoader.resolve (node:internal/modules/esm/loader:251:12)
    at DefaultModuleLoader.getModuleJob (node:internal/modules/esm/loader:140:32)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:33)
    at link (node:internal/modules/esm/module_job:75:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v20.5.1
```

Any help is appreciated!

**Comment by smikitky** [comment]
Thank you very much. I was importing the module incorrectly. I hadn't noticed the bug because I was testing this mainly using `tsx`. I should have done a final check with the built script...

I just released v1.7.1. Could you please check if the issue has been fixed?

**Comment by deining** [comment]
> Thank you very much. I was importing the module incorrectly. I hadn't noticed the bug because I was testing this mainly using `tsx`. I should have done a final check with the built script...
> 
> I just released v1.7.1. Could you please check if the issue has been fixed?

Thanks for your quick action! With version v1.7.1, I can now run `chatgpt-md-translator` without problems.
